### PR TITLE
[MODULAR] Puts waterbreathing on the species quirks page, adds a waterbreathing quirk

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request

Title. 

## How This Contributes To The Skyrat Roleplay Experience

1. It was undocumented before, so while it  could be inferred, it wasn't actually stated that aquatics could breathe underwater.

2. Waterbreathing is sufficiently generic of a thing to be given to anything. You can breathe water for a lot of reasons, and it is hardly an impactful effect whatsoever. The only times it comes into play is when you're, well, under water, which is very rare.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
  
image

Its scary how good of a coder I am
  
</details>

## Changelog

:cl:
add: A waterbreathing quirk
qol: Waterbreathing is now documented on species pages of the species that have it
/:cl:

### CODE REVIEW

**Comment by Jolly-66** [comment]
I don't know why I thought this said "waterbathing"

File: modular_skyrat/master_files/code/datums/traits/good.dm [hunk 1]
```diff
@@ -68,6 +68,16 @@
...
 		right_arm.unarmed_sharpness = initial(right_arm.unarmed_sharpness)
 
+/datum/quirk/water_breathing
+	name = "Water breathing"
+	desc = "You are able to breathe underwater!"
+	value = 2
+	mob_trait = TRAIT_WATER_BREATHING
+	gain_text = span_notice("You feel the moisture of the air in your lungs.")
+	lose_text = span_danger("You feel normally wary around large bodies of water!")
+	medical_record_text = "Patient possesses biology compatable with aquatic respiration."
```
Comment (vinylspiders) [inline modular_skyrat/master_files/code/datums/traits/good.dm]:
```suggestion
	medical_record_text = "Patient possesses biology compatible with aquatic respiration."
```

File: modular_skyrat/master_files/code/datums/traits/good.dm [hunk 2]
```diff
@@ -68,6 +68,16 @@
 		right_arm.unarmed_miss_sound = initial(right_arm.unarmed_miss_sound)
 		right_arm.unarmed_sharpness = initial(right_arm.unarmed_sharpness)
 
+/datum/quirk/water_breathing
+	name = "Water breathing"
+	desc = "You are able to breathe underwater!"
+	value = 2
+	mob_trait = TRAIT_WATER_BREATHING
+	gain_text = span_notice("You feel the moisture of the air in your lungs.")
```
Comment (vinylspiders) [inline modular_skyrat/master_files/code/datums/traits/good.dm]: Maybe you could tweak this to make it clearer that it's a positive, satisfactory feeling. So it's a lil clearer what's going on.

File: modular_skyrat/master_files/code/datums/traits/good.dm [hunk 3]
```diff
@@ -68,6 +68,16 @@
...
+	lose_text = span_danger("You feel normally wary around large bodies of water!")
```
Comment (vinylspiders) [inline modular_skyrat/master_files/code/datums/traits/good.dm]: For symmetry, might be better if you kept the description along the same theme as the `gain_text` (aka, something along the lines of no longer feeling satisfaction from breathing moisture).

**Review by vinylspiders — CHANGES_REQUESTED** [review]
Otherwise looks good to me. A good addition!

File: modular_skyrat/master_files/code/datums/traits/good.dm (hunk 3 above)
Comment (nikothedude) [reply modular_skyrat/master_files/code/datums/traits/good.dm]: fair

**Review by vinylspiders — APPROVED** [review]

**Pull request merged**